morango/settings.py

```diff
@@ -1,18 +1,22 @@
 # Django settings for morango project.
+import os
```

Comment:
Does this settings file really belong in the morango root, or within the testapp?